src/common/obj_bencher.cc

```diff
@@ -605,10 +615,15 @@ int ObjBencher::seq_read_bench(int seconds_to_run, int num_objects, int concurre
     lock.Unlock();
     release_completion(slot);
     cur_contents = contents[slot];
-
+    if (!cur_contents) {
+        contents[slot] = new bufferlist();
+        cur_contents = contents[slot];
+    }
+    // hacky way to invalidate internal crc cache
+    cur_contents->zero(0, 0);
+        
```

Comment:
This should not be necesssary unless you have evidence that **aio_read** modifies cur_contents without using the bufferlist API ?